SetUp.ps1

```diff
@@ -36,7 +36,7 @@ param(
     [Parameter(Mandatory = $false, HelpMessage = 'The version of OPA Rego to be downloaded, must be in "x.x.x" format')]
     [Alias('version')]
     [string]
```

Comment:
```suggestion
    [version]
```

Reply:
I am not modifying that workflow. Please open a new issue and let whoever is in charge of that workflow know that it should be changed.